x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/Downsample.java

```diff
@@ -45,6 +45,7 @@ public class Downsample extends Plugin implements ActionPlugin, PersistentTaskPl
 
     public static final String DOWSAMPLE_TASK_THREAD_POOL_NAME = "downsample_indexing";
     private static final int DOWNSAMPLE_TASK_THREAD_POOL_QUEUE_SIZE = 256;
+    public static final String DOWNSAMPLE_MIN_NUMBER_OF_REPLICAS = "downsample.min_number_of_replicas";
```

Comment:
Is this the right place for this constant? Are we able to reference to this in serverless

Reply:
My understanding is that this is not important. As you see here it is just a string that happens to identify a Setting which only exists in Serverless deployments. So for Stateful deployments that string will result in no setting being found and the default value being used.

Reply:
ok 👍